docs/swagger2.yml

```diff
@@ -313,9 +327,9 @@ definitions:
       image_id:
         type: string
         description: "docker image id, sha hash"
-      revision:
+      revision_id:
```

Comment:
```
revision:
  type: int
```

seems inline w/ what we've discussed, keeping it a string here in case we change our minds about monotonic int?